pilot/pkg/networking/core/v1alpha3/envoyfilter/listener_patch.go

```diff
@@ -63,7 +63,7 @@ func ApplyListenerPatches(
 
 func patchListeners(
 	patchContext networking.EnvoyFilter_PatchContext,
-	envoyFilterWrapper *model.EnvoyFilterWrapper,
+	efw *model.EnvoyFilterWrapper,
```

Comment:
Nit: probably pass in only efw.Key() instead of the whole wrapper? Same to all other places Seems cleaner and easier to interpret the code in that way.